<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SFSensorControlWidget</class>
 <widget class="QWidget" name="SFSensorControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutAccelerationX" stretch="10,1">
     <item>
      <widget class="QSlider" name="accelerationX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="accelerationXState"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutAccelerationY" stretch="10,1">
     <item>
      <widget class="QSlider" name="accelerationY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="accelerationYState"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutAccelerationZ" stretch="10,1">
     <item>
      <widget class="QSlider" name="accelerationZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="accelerationZState"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRun">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
